cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
```

Comment:
Ah, I had no idea slugs were not unique - thanks @rosskarchner and @chosak for pointing that out. What if we could do it either by slug or by ID? For the majority of cases the former would work, and when that fails, the person would just have to go look up the ID for the one they want.